crates/meta/src/storage.rs

```diff
@@ -8,13 +8,23 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+use codederror::CodedError;
 use restate_schema_impl::SchemasUpdateCommand;
 use serde::{Deserialize, Serialize};
 use std::future::Future;
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
 use tokio::io;
 use tracing::trace;
 
+type StorageFormatVersionType = u32;
+
+/// Storage format version used by the [`FileMetaStorage`] to store schema information. This value
+/// must be incremented whenever you introduce a breaking change to the schema information.
+const STORAGE_FORMAT_VERSION: StorageFormatVersionType = 0;
+
+/// Name of the file which contains the storage format version.
+const STORAGE_FORMAT_VERSION_FILE_NAME: &str = ".version";
```

Comment:
Can we give this a meta-specific name?  e.g. `.meta_format_version`

Reply:
jup. Will change it.